pkg/apis/servicecatalog/fake/pointers.go

```diff
@@ -0,0 +1,5 @@
+package fake
+
+func boolPtr(b bool) *bool {
+	return &b
```

Comment:
Having never actually tried it, I tried it, and apparently you can't take the address of a bool. Bizzare language.

Would it be better to have it once in a toplevel util package than rewriting this in a multiple places.